CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(crow-translate
 )
 
 set(APPLICATION_NAME "Crow Translate")
+set(APPLICATION_ID "io.crow_translate.CrowTranslate")
```

Comment:
Could you reuse this variable later in this file and in C++ code by adding it to https://github.com/crow-translate/crow-translate/blob/master/src/cmake.h.in?